composer.json

```diff
@@ -7,14 +7,17 @@
     "require": {
         "composer/installers": "~1.2",
         "tecnickcom/tcpdf": "^6.2",
-        "stripe/stripe-php": "^6.41"
+        "stripe/stripe-php": "^6.41",
+        "paypal/paypal-checkout-sdk": "1.0.1"
     },
     "require-dev": {
         "squizlabs/php_codesniffer": "*",
         "dealerdirect/phpcodesniffer-composer-installer": "^0.6",
         "phpcompatibility/php-compatibility": "^9.3",
         "phpcompatibility/phpcompatibility-wp": "*",
-        "wp-coding-standards/wpcs": "*"
+        "wp-coding-standards/wpcs": "*",
+        "jetbrains/phpstorm-stubs": "dev-master",
+        "humbug/php-scoper": "dev-master"
```

Comment:
Please don't use "dev-master" unless absolutely necessary. We can use "^0.13" instead.

Reply:
fixed